tests/llmobs/test_llmobs_service.py

```diff
@@ -678,89 +617,69 @@ def test_annotate_incorrect_document_field_type_raises_warning(LLMObs, mock_logs
 def test_annotate_output_string(LLMObs):
```

Comment:
imo a bunch of these tests are just asserting implementation details which makes changing the underlying storage mechanism for the data harder to change. I think I'd be ok with getting rid of them all so long as we have good coverage at a higher level testing the real use-cases. WDYT @Yun-Kim?

Reply:
Yeah I think that's a good idea - thanks!

Reply:
Gonna defer that to a follow-up PR if that's alright with you - I have an idea to assert the generated LLMObs span event rather than the internal APM span store object implementation, but this will probably be a bigger change than in the context of this PR.

Reply:
over my dead body